Tasks/DotNetCoreCLIV2/Tests/L0.ts

```diff
@@ -13,6 +13,7 @@ describe('DotNetCoreExe Suite', function () {
     });
 
     it('fails if the dotnet tool is not found', (done: MochaDone) => {
+        this.timeout(5000);
```

Comment:
is this needed?

Reply:
updated.Default timeout is 2000ms .Sometimes after test fails because mocha done() function is not called within this time for test.